### PR TITLE
Fix test failure when cudnn is unavailable

### PR DESCRIPTION
**Pull request by kmaehashi**
Fix issue introduced in #1885.

### CODE REVIEW

**Comment by toslunar** [comment]
Jenkins, test this please.

File: tests/cupy_tests/test_cudnn.py [hunk 1]
```diff
@@ -14,6 +15,10 @@
     coef_modes = [
         libcudnn.CUDNN_ACTIVATION_CLIPPED_RELU,
     ]
+    layouts = [
+        libcudnn.CUDNN_TENSOR_NCHW,
+        libcudnn.CUDNN_TENSOR_NHWC,
+    ],
```
Comment (toslunar) [inline tests/cupy_tests/test_cudnn.py]:
```suggestion
    ]
```

**Comment by kmaehashi** [comment]
Jenkins, test this please.

**Comment by toslunar** [comment]
Jenkins, test this please.

**Comment by toslunar** [comment]
The Jenkins failures are unrelated to the PR.

**Comment by toslunar** [comment]
#1885 was merged in v7 and was not backported.

**Pull request merged**